packages/cdktf-cli/bin/cmds/ui/models/terraform.ts

```diff
@@ -49,6 +49,10 @@ export interface TerraformOutput {
   value: string | Record<string, unknown> | Array<any>;
 }
 
+export function isTerraformOutput(output: any): output is TerraformOutput {
+  return typeof output.value === "string";
```

Comment:
What if `value` is `Record<string, unknown>` or `Array<any>` as allowed by the type definition?